components/eamxx/src/physics/mam/mam_coupling.hpp

```diff
@@ -522,6 +523,7 @@ haero::Atmosphere atmosphere_for_column(const DryAtmosphere& dry_atm,
                            ekat::subview(dry_atm.ni, column_index),
                            ekat::subview(dry_atm.z_mid, column_index),
                            ekat::subview(dry_atm.p_del, column_index),
+			   interface_pressure,
```

Comment:
I will have to look at the code again but it will be nice if we keep the format the same. That is, we take a subview of `dry_atm.p_int` (or whatever the variable is called) instead of using `interface_pressure`.

Reply:
FIxed in 76ca0876a255864e5aa425144524aa62fed2d1ba where the pressure at interfaces is added to the DryAtmosphere structure.